### DISCUSSION TITLE
99% statistic by countries - Unknown

### DISCUSSION THREAD

**Discussion by garayev** [umami-software/umami#785] [Q&A]
99% statistics by countries - is Unknown country. I think some GeoIP is working incorrectly. What do I need to do to make statistics work? Umami 1.22

**Comment by viharm** [comment]
I noticed that this appears to be in the code https://github.com/umami-software/umami/blob/c7275ef85b124ed88126877487ff1c9fddaf068e/scripts/build-geo.js#L19
Is it released?
